src/AltinnCore/Runtime/Startup.cs

```diff
@@ -161,6 +161,7 @@ public void ConfigureServices(IServiceCollection services)
                         ValidateLifetime = true
                     };
                     options.ExpireTimeSpan = new TimeSpan(0, 30, 0);
```

Comment:
The cookie should be a session cookie. Now it will exist after the browser is closed. Should not have a time out. Agree @alt-how ?

Reply:
This cookie should be a session cookie since the token inside expires.

Reply:
Yes, in the browser i see that it is a session cookie, @TheTechArch do you mean any other property specific to  session cookie?

Reply:
Removed the time out